### ISSUE TITLE
epd7in3f support？

### ISSUE THREAD

**Issue by cloud263** [txoof/PaperPi#86]
May I ask whether it supports epd7in3f now, which is Waveshare's new ink screen.

**Comment by txoof** [comment]
@cloud263 Yes! The development branch supports all the f screens. I need a few more people to test it out and then I can merge the changes into the production version. If you have some time to test it out and give some feedback, I would greatly appreciate it!

You can join the discussion here: https://github.com/txoof/PaperPi/discussions/77

And you can install the development branch using this curl installer: 

```
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/txoof/PaperPi/I%2379_color_display/install/remote_install_development.sh)"
```

**Comment by txoof** [comment]
@cloud263 Any luck trying the development version?
